packages/default-theme/components/SwProductDetails.vue

```diff
@@ -156,12 +154,19 @@ export default {
     }
   },
   computed: {
-    price() {
-      return getProductRegularPrice({ product: this.product })
+    getPrice() {
+      // remove that logic once the SW6 API returns right data
+      // related: https://github.com/DivanteLtd/shopware-pwa/issues/263
+      const regularPrice = getProductRegularPrice({ product: this.product })
+      const specialPrice = getProductSpecialPrice(this.product)
+      return regularPrice > specialPrice ? regularPrice : specialPrice
     },
     getSpecialPrice() {
-      const price = getProductSpecialPrice(this.product)
-      return price && '$' + price
+      // remove that logic once the SW6 API returns right data
+      // related: https://github.com/DivanteLtd/shopware-pwa/issues/263
+      const regularPrice = getProductRegularPrice({ product: this.product })
+      const specialPrice = getProductSpecialPrice(this.product)
+      return regularPrice > specialPrice ? specialPrice : regularPrice
```

Comment:
Should it works like that? :D I know that it's strange that special price is higher than regular, but it was set as special 🤔

Reply:
see the comment above :P will be deleted as soon as we have it on API side fixed ;)